src/services/me/docs/openapi.yaml

```diff
@@ -12,57 +12,177 @@ info:
     name: GPL-3.0
     url: 'https://github.com/hpi-schul-cloud/schulcloud-server/blob/master/LICENSE'
   version: 1.0.0
+
 components:
   securitySchemes:
     jwtBearer:
       type: http
       scheme: bearer
       bearerFormat: JWT
   schemas:
-    me:
-      description: TODO
-    me_list:
-      description: TODO
+    consent:
+      type: object
+      properties:
+        userConsent:
+          type: object
+          properties:
+            form:
```

Comment:
Is there a way to specify a enum? As far as I know, the form attribute have to be one of theses strings: `['analog', digital', 'update']`